db/commit.go

```diff
@@ -49,6 +51,7 @@ type Local struct {
 	Date       datetime.DateTime
 	Name       string
 	Args       types.Value
+	Original   hash.Hash
```

Comment:
[fix?] I don't understand why/how this gets saved into Noms. AFAIK in order for this field to get marshaled to Noms, there'd have to be a method defined for `hash.Hash` that tells Noms how to store it. I don't see one in Noms. It seems like marshal should panic unless there's a custom marshaler defined for `Local` that handles this, which I also don't see. But clearly you're testing this ... so?

Reply:
<img width="805" alt="Screen Shot 2020-05-04 at 9 29 59 PM" src="https://user-images.githubusercontent.com/80388/81043751-9bc20b00-8e4e-11ea-8784-41d39c266155.png">

![ruh-roh](https://user-images.githubusercontent.com/80388/81044188-8ef1e700-8e4f-11ea-81d1-e827ddb1308c.jpg)

<img width="352" alt="Screen Shot 2020-05-04 at 9 30 45 PM" src="https://user-images.githubusercontent.com/80388/81044226-9fa25d00-8e4f-11ea-9502-99120ff2780c.png">